corehq/tests/noseplugins/uniformresult.py

```diff
@@ -32,6 +33,8 @@ def uniform_description(test):
             descriptor.__name__,
             test.arg,
         )
+    if isfunction(test):
+        return test.__name__
```

Comment:
@millerdev the build was failing here on `test_on_starting`. Not sure why `test_on_starting` is not of type `FunctionTestCase`. Does this seem like a reasonable solution.
```
======================================================================
ERROR: test suite for <function test_on_starting at 0x7f9820345680>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/skelly/.virtualenvs/hq/lib/python3.7/site-packages/nose/suite.py", line 210, in run
    self.setUp()
  File "/home/skelly/.virtualenvs/hq/lib/python3.7/site-packages/nose/suite.py", line 293, in setUp
    self.setupContext(ancestor)
  File "/home/skelly/.virtualenvs/hq/lib/python3.7/site-packages/nose/suite.py", line 302, in setupContext
    self.config.plugins.startContext(context)
  File "/home/skelly/.virtualenvs/hq/lib/python3.7/site-packages/nose/plugins/manager.py", line 99, in __call__
    return self.call(*arg, **kw)
  File "/home/skelly/.virtualenvs/hq/lib/python3.7/site-packages/nose/plugins/manager.py", line 167, in simple
    result = meth(*arg, **kw)
  File "/home/skelly/src/cchq/corehq/tests/noseplugins/timing.py", line 87, in startContext
    self.end_event("before", context)
  File "/home/skelly/src/cchq/corehq/tests/noseplugins/timing.py", line 64, in end_event
    name = uniform_description(context)
  File "/home/skelly/src/cchq/corehq/tests/noseplugins/uniformresult.py", line 39, in uniform_description
    test._testMethodName
AttributeError: 'function' object has no attribute '_testMethodName'
```

Reply:
I agree, nose should have wrapped it in `FunctionTestCase` before it gets to this point in the code.

Your fix seems reasonable, but I would also include the module name
```suggestion
        return "%s:%s" % (test.__module__, test.__name__)
```
Alternately, I think line 27 could be changed to
```py
if isinstance(test, type) or isfunction(test):
```